common/src/main/java/com/wynntils/core/text/StyledText.java

```diff
@@ -629,10 +633,7 @@ public Iterator<StyledTextPart> iterator() {
 
     @Override
     public String toString() {
-        return "StyledText{'" + getString() + "', parts="
-                + ArrayUtils.toString(parts) + ", clickEvents="
-                + ArrayUtils.toString(clickEvents) + ", hoverEvents="
-                + ArrayUtils.toString(hoverEvents) + '}';
+        return "StyledText{'" + getString() + "'}";
```

Comment:
Perhaps getString(INCLUDE_EVENTS) would be the best here.

Reply:
Good idea. I forgot about it.